test/integration/windows/windows_suite_test.go

```diff
@@ -51,6 +54,10 @@ var _ = BeforeSuite(func() {
 	ctx = context.Background()
 	verify = verifier.NewPodVerification(frameWork, ctx)
 
+	By("creating the configmap with enable-windows-ipam set to true")
+	configMap = manifest.NewConfigMapBuilder().Build()
+	configMapWrapper.CreateConfigMap(frameWork.K8sClient, ctx, configMap)
```

Comment:
If the ConfigMap is already there will this fail? I think we can do 
- Get
  - If present, enable the feature
  - Create otherwise

Reply:
Will resolve this in a follow up PR.

Reply:
Resolved in the recent commit to fix integration test errors.